SplashScreen/app/src/main/java/com/example/android/splashscreen/CustomActivity.kt

```diff
@@ -36,27 +34,29 @@ class CustomActivity : MainActivity() {
 
         // This callback is called when the app is ready to draw its content and replace the splash
         // screen. We can customize the exit animation of the splash screen here.
-        splashScreen.setOnExitAnimationListener { splashScreenView ->
+        installedSplashScreen.setOnExitAnimationListener { splashScreenViewProvider ->
 
             // The animated vector drawable is already animating at this point. Depending on the
             // duration of the app launch, the animation might not have finished yet.
             // Check the extension property to see how to calculate the remaining duration of the
             // icon animation.
-            val remainingDuration = splashScreenView.iconAnimationRemainingDurationMillis
+            val remainingDuration = splashScreenViewProvider.iconAnimationRemainingDurationMillis
 
-            // The callback gives us a `SplashScreenView` as its parameter. This is the view for the
-            // entire splash screen.
+            // The callback gives us a `SplashScreenViewProvider` as its parameter. It holds the
+            // view for the entire splash screen.
+            val view = splashScreenViewProvider.view
             val slideUp = ObjectAnimator.ofFloat(
-                splashScreenView,
+                view,
```

Comment:
You can keep the name splashScreenView. I think it's more explicit and easier to follow.

Reply:
Done.